wcfsetup/install/files/lib/system/package/PackageInstallationDispatcher.class.php

```diff
@@ -350,11 +355,46 @@ public function install($node) {
 				WHERE		processNo = ?";
 			$statement = WCF::getDB()->prepareStatement($sql);
 			$statement->execute([$this->queue->processNo]);
+			
+			$this->logInstallationStep([], 'finished cleanup');
 		}
 		
 		return $step;
 	}
 	
+	/**
+	 * Logs an installation step.
+	 * 
+	 * @param	array		$node	data of the executed node
+	 * @param	string		$log	optional additional log text
+	 */
+	protected function logInstallationStep(array $node = [], $log = '') {
+		$time = TIME_NOW;
+		$logEntry = "[" . TIME_NOW . "]\n";
```

Comment:
Consider using `time()` here, the update process might take longer than a second.

Reply:
I am unsure if that information is really useful in practice. The goal of this change in general is just to have a log of the last step when a problem occurs during an update so that the _order_ of the entries is of relevance.